src/components/HarborRepository/tableHeadings.tsx

```diff
@@ -54,7 +54,7 @@ export const columns: TableColumn[] = [
     field: 'repoUrl',
     render: (rowData: any) => (
       <Button
-        to={`${rowData.repoUrl}/artifacts/${rowData.tag}`}
+        to={`${rowData.repoUrl}/artifacts-tab/artifacts/${rowData.tag}`}
```

Comment:
this url take a digest in param and not a tag ( see api of harbor)

Reply:
I have tested the same fix and it takes a tag. Is it maybe a different harbor version?

Reply:
Both, the artifact digest, as well as the artifact name seem to work for me

![image](https://github.com/container-registry/backstage-plugin-harbor/assets/32169493/e0f825a1-98a2-4c99-b484-fbed0cb8f03e)

The artifact name is being returned as `tag` by the backend plugin

https://github.com/container-registry/backstage-plugin-harbor-backend/blob/master/src/service/artifact.ts#L46-L48

https://github.com/container-registry/backstage-plugin-harbor-backend/blob/master/src/service/artifact.ts#L109